File: Java/EX05.java
import javax.swing.JOptionPane;

public class EX05{
    public static void main(String[] args){
        int n1;

        n1 = Integer.parseInt(JOptionPane.showInputDialog(null,"Digite um número:"));

       for(int a = 0, b = 1, i = 0; i < n1; b += a, a = b-a, i++) {
			System.out.print(a + " ");
		}  
    }
}

File: C/EX05.c
#include <stdio.h>

int main(){
    int n1;

    printf("Digite um número:");
    scanf("%d",&n1);

    for (int a = 0, b = 1 , i = 0;i < n1;b = b + a, a = b - a,i++){
        printf("%d ",a);
    }

    return 0;
}